### PR TITLE
fix: set metadata columns types

### PR DESCRIPTION
**Pull request by TyShkan**
Closes https://github.com/MeltanoLabs/tap-csv/issues/131

### CODE REVIEW

**Comment by TyShkan** [comment]
@pnadolny13 could you please take a look?

File: tap_csv/client.py [hunk 1]
```diff
@@ -134,18 +134,22 @@ def schema(self) -> dict:
...
                 SDC_SOURCE_FILE_COLUMN,
                 SDC_SOURCE_FILE_MTIME_COLUMN,
                 SDC_SOURCE_LINENO_COLUMN,
             ] + header
-
-        for column in header:
-            # Set all types to string
-            # TODO: Try to be smarter about inferring types.
-            properties.append(th.Property(column, th.StringType()))
+
```
Comment (pnadolny13) [inline tap_csv/client.py:149]:
```suggestion

            properties.append(th.Property(SDC_SOURCE_FILE_COLUMN, th.StringType))
            properties.append(
                th.Property(SDC_SOURCE_FILE_MTIME_COLUMN, th.DateTimeType)
            )
```

@TyShkan linting was complaining about the formatting of this section. I think it this should fix it or you can run `poetry run tox -e format`.

**Review by pnadolny13 — COMMENTED** [review]
@TyShkan I agree with this, good idea! I left a comment related to reformatting but other than that this looks good to me.

**Review by pnadolny13 — APPROVED** [review]
@TyShkan this is great!

**Comment by TyShkan** [comment]
Oops, seems like I let my local poetry.toml to escape! :]

**Pull request merged**